### PR TITLE
Hackathon 2022

### PR DESCRIPTION
**Pull request by LimaoC**
Update website for hackathon 2022

### CODE REVIEW

**Review by JamesDearlove — APPROVED** [review]
Looks mostly good to me, just a typo I found and a comment.

Nit: I don't really care, but it has been brought to my attention that there is an inconsistent use of `-` and `&ndash;` on the page. With that said, it's exactly like that in the previous Hackathon pages.

File: content/competitions/hackathon-2022.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            <p>The creators of the best project (as determined by our celebrity judges) will take home a whopping $2000
+                dollars (not to mention bragging rights)! This prize is proudly sponsored by TechnologyOne.
+            </p>
+
+            <h5>$1000 - Best Overall Runner Up</h5>
+            <p>The second placed team (as determined by our celebrity judges) will take home $1000!
+            </p>
+
+            <h5>$500 - Best Newcomers</h5>
+            <p>The best team consisting entirely of students in their first or second semester of university (in any discipline) will find themselves the proud owners of $500 to distribute between themselves.
```
Comment (andrewj-brown) [inline content/competitions/hackathon-2022.html:67]: Perhaps a note that this isn't hard-and-fast and the final decision of whether a team is eligible for the first-year-prize will be made by committee on the day?

Comment (andrewj-brown) [reply content/competitions/hackathon-2022.html:67]: Also "Best Newcomers" is a good title but it sorta implies new-to-this, not "first-year"

**Review by andrewj-brown — COMMENTED** [review]
One comment on Best Newcomers (first-year-prize) but other than that :lgtm:

File: content/competitions/hackathon-2022.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+			<p>
+                Form your teams, brainstorm your cool ideas, and get ready to eat tons of free food... UQCS Hackathon 2022 is here!! Featuring prizes bigger and better than ever, SEVEN free meals, and 24 hour stay (TBD)! That's right - you'll be able to stay at uni overnight to get that authentic hackathon experience (TM).
+			</p>
+
+			<p>
+                Any year and experience level is welcome! In the past, students have built websites, desktop apps, games, hardware, and even robots and if you've never been to a hackathon before, not to worry - we'll be hosting a talk before hackathon weekend to walk you through and give you tips and tricks for your first hackathon experience.
+			</p>
+
+			<p>
+                <b>EARLY BIRD TICKETS</b>: student tickets will only be $10 if you purchase one <b>on or before 5th Aug</b>, or only $15 with a <i>limited edition laptop blahaj</i> shirt included! Other ticket types can be found either below or at the Humanitix event link.
```
[the remark this answers is not in the document]
Comment (LimaoC) [reply content/competitions/hackathon-2022.html:35]: Ah I was referring to the different ticket types described below in the Pricing section but on a second read it is a bit confusing, I'll just change it to

`Other ticket types can be found at the Humanitix event link above.`

File: content/competitions/hackathon-2022.html (hunk 1 above)
Comment (LimaoC) [reply content/competitions/hackathon-2022.html:67]: Fair point, will change to "Best First-Years"

Comment (bradleysigma) [reply content/competitions/hackathon-2022.html:67]: _Why not "Best Neophytes"?_

File: content/competitions/hackathon-2022.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (JamesDearlove) [reply content/competitions/hackathon-2022.html]: Sounds good

**Review by Thomas-Malcolm — APPROVED** [review]
All looks good - ready for merge!

**Pull request merged**